client/src/components/Lists/Users.js

```diff
@@ -25,35 +25,12 @@ import PersonIcon from '@material-ui/icons/Person';
 import RemoveIcon from '@material-ui/icons/Remove';
 import Avatar from '@material-ui/core/Avatar';
 import { shape, string } from 'prop-types';
-//import userListSelector,{chartOperations} from '../../state/redux/charts';
 import { authSelectors, authOperations } from '../../state/redux/auth';
 import { userListType, getUserListType } from '../types';
 const styles = theme => {
 	const { type } = theme.palette;
 	const dark = type === 'dark';
 	return {
-		container: {
-			width: 'auto',
-			display: 'block', // Fix IE 11 issue.
-			marginLeft: theme.spacing.unit * 3,
-			marginRight: theme.spacing.unit * 3,
-			[theme.breakpoints.up(400 + theme.spacing.unit * 3 * 2)]: {
-				width: 400,
-				marginLeft: 'auto',
-				marginRight: 'auto'
-			}
-		},
-		paper: {
-			display: 'flex',
-			flexDirection: 'column',
-			alignItems: 'center',
-			padding: `${theme.spacing.unit * 2}px ${theme.spacing.unit * 3}px ${theme
-				.spacing.unit * 3}px`
-		},
-		form: {
-			width: '100%', // Fix IE 11 issue.
-			marginTop: theme.spacing.unit
-		},
```

Comment:
I'm not sure it's OK to delete this part. Your thought? @JeevaSang @MekiaE 
What is the reason deleting this part?
I have same question to across the files modified with this kind of code deletion for fixing code duplication alert regarding styles in HOC(Higher Order Component) .

Reply:
I have deleted the unused style classes. Added container and paper style classes back